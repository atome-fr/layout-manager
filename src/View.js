import React from 'react';

/**
 * Method for render the component
 * @param {Object} props The props passed to the component
 */
function View(props) {
    return (
        <div className="component">
            {props.children}
        </div>
    );
}

export default View;