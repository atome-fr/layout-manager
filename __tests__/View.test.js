import React from 'react';
import View from '../src/View';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('View Component', () => {

    describe('UI Render Test', () => {

        it('should render the View component', () => {
            const wrapper = shallow(<View/>);
            const tree = toJson(wrapper);

            expect(tree).toMatchSnapshot();
        });

    });

});