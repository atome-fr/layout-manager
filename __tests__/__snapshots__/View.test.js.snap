// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Component UI Render Test should render the View component 1`] = `
<div
  className="component"
/>
`;
